backend/app/interior/interface/controller/interior_controller.py

```
@@ -0,0 +1,55 @@
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)